### ISSUE TITLE
Migrate content for Search release

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#169]
This issue will track the redirection and creation of files to support the initial search release.

**Comment by humancompanion** [comment]
[USAJOBS-help-center-content-for-search_20170217.xlsx](https://github.com/USAJOBS/Help/files/784511/USAJOBS-help-center-content-for-search_20170217.xlsx)

@dalenard21 - I've updated the spreadsheet with the changes I made and a few new pages:

1. /search/sort/ wasn't a separate page. We handled it on the understanding search results page. So that content is duplicated now but, that's probably okay.
2. /search/location/ didn't exist but /search/keyword/ does so it made sense to me to create it even though we also have /search/filters/location/ which should contain Distance within (formerly radius) and Relocation assistance.
